graphql/src/main/java/com/scalar/db/graphql/server/GraphQlServer.java

```diff
@@ -0,0 +1,60 @@
+package com.scalar.db.graphql.server;
+
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.service.StorageFactory;
+import java.io.FileInputStream;
+import java.util.Properties;
+import java.util.concurrent.Callable;
+import org.eclipse.jetty.server.Server;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+@Command(name = "scalardb-graphql", description = "Starts Scalar DB GraphQL server.")
+public class GraphQlServer implements Callable<Integer> {
+  @CommandLine.Option(
+      names = {"--properties", "--config"},
+      required = true,
+      paramLabel = "PROPERTIES_FILE",
+      description = "A configuration file in properties format.")
+  private String configFile;
+
+  private ServerConfig config;
+
+  public static void main(String[] args) {
+    int exitCode = new CommandLine(new GraphQlServer()).execute(args);
+    System.exit(exitCode);
+  }
+
+  @Override
+  public Integer call() throws Exception {
+    if (configFile != null) {
+      Properties properties = new Properties();
+      try (FileInputStream fis = new FileInputStream(configFile)) {
+        properties.load(fis);
+      }
+      config = new ServerConfig(properties);
+    }
```

Comment:
It looks like configFile is always not null? And we can use the constructor `ServerConfig(File propertiesFile)` here?

```suggestion
  config = new ServerConfig(new File(configFile));
```

Reply:
`configFile` can never be null as long as it is invoked via CLI, but an additional constructor that accepts a `ServerConfig` instance will be needed for integration testing (not yet implemented). I added the constructor in advance just like [ScalarDbServer](https://github.com/scalar-labs/scalardb/blob/graphql-server/server/src/main/java/com/scalar/db/server/ScalarDbServer.java) in 9c1cedc.